.github/workflows/push.yml

```diff
@@ -0,0 +1,115 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+# https://github.com/actions/setup-ruby
+# https://github.com/ruby/setup-ruby#versioning
+
+
+name: CI 
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Run test suite
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        ruby: [ '2.6' ] 
+        node: [ '12', '14' ] 
+
+    # use service instead of docker-compose as the `bundle exec rake` command installs Solr & Fedora
+    #env:
+    #  COMPOSE_FILE: docker-compose.lightweight.yml
```

Comment:
There are two options to setup the infrastructure to run the tests: via the services pattern or via docker-compose (like Jupiter). This was a note to future to show how to adapt the GitHub action to the docker-compose pattern. However, since the move away from Avalon, I've removed the comments (and added a note at the top).